cli/src/main/resources/help/ls.adoc

```diff
@@ -13,6 +13,24 @@ SYNOPSIS
 *ls* ['PATH']
 
 
+OPTIONS
+-------
+
+*-f, --format*='[NAMES|VALUES|FULL]'::
+This option only applies when listing caches.
+* `NAMES`: only show the keys
+* `VALUES`: show the keys and values
+* `FULL`: show keys, values and metadata
+
```

Comment:
you need an empty line otherwise, everything is rendered in the same line.

Reply:
fixed